ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/SlidingWindowRateLimiter.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.server;
+
+import static io.confluent.ksql.util.KsqlPreconditions.checkArgument;
+
+import io.confluent.ksql.util.KsqlException;
+import io.confluent.ksql.util.Pair;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class SlidingWindowRateLimiter {
+  private static long NUM_BYTES_IN_ONE_MEGABYTE = 1048576;
```

Comment:
It's a bit easier to read if you write this as `1 * 1024 * 1024` or something like that.  The compiler will obviously turn it into the above, but it makes it easier to change the constant.